packages/app/app/containers/BestNewMusicCardContainer/index.tsx

```diff
@@ -29,19 +17,20 @@ function mapStateToProps(state, ownProps) {
   };
 }
 
-function mapDispatchToProps(dispatch) {
-  return {
-    actions: bindActionCreators(
-      Object.assign(
-        {},
-        FavoritesActions,
-        ToastActions
-      ),
-      dispatch
-    )
-  };
+export interface BestNewMusicCardContainerProps extends ActionCreatorsMapObject {
+  FavoritesActions: ActionCreator<any>;
+  TostActions: ActionCreator<any>;
 }
 
+const mapDispatchToProps = (dispatch) => ({actions: bindActionCreators<BestNewMusicCardContainerProps, any>(
+  Object.assign(
```

Comment:
It's better to use spread (`{...stuff}`) instead of `Object.assign` now. I didn't know it existed when I wrote this (or maybe it wasn't available yet).

Reply:
Have made this change